src/components/Dropdown.vue

```diff
@@ -40,26 +40,35 @@
                 :is="item.component"
                 :active="active"
               />
+              <div v-else-if="item.divider">
+                <hr class="m-1 bg-gray-300" />
+              </div>
```

Comment:
m-1 applies margin on all sides? you probably want margin only on y-axis. Check Figma design. Also, use semantic class for bg, `bg-surface-gray-*` instead of `bg-gray-`